system/spd.c

```diff
@@ -0,0 +1,978 @@
+// SPDX-License-Identifier: GPL-2.0
+// Copyright (C) 2004-2023 Sam Demeulemeester
+
+#include "display.h"
+
```

Comment:
My aim was to have a clean separation between the application code and the low-level libraries, with the former depending on the latter, but not vice-versa. So I don't like the inclusion of `display.h` here, and would like to move the display code into the `app` directory. But that can be another step if you don't want to do it now.

Reply:
Understood. We can limit this to` #include "print.h"` for now. Longer term, perhaps both print_spd_startup_info() and print_spdi() should go to a new place. Once LA64 changes land, I will think about a good way to do it.